lib/apiservers/engine/backends/container.go

```diff
@@ -117,6 +119,38 @@ const (
 	DefaultEnvPath         = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
 )
 
+func NotFoundError(msg string) error {
+	return derr.NewRequestNotFoundError(fmt.Errorf("No such container: %s", msg))
+}
+
+func UnknownError(msg string) error {
```

Comment:
The original code was using those, the only difference is the string that we are returning. Server error from vs Unknown error from. I could remove Unknown error and use ServerError everywhere if Unknown error messages are not required for some reason